MACOS_SETUP.md

```diff
@@ -0,0 +1,166 @@
+# Local DP Dash development on Mac
+
+This document outlines the set up necessary to run the application on a Mac computer.
+
+
+## Requirements
+  * [Completed all the steps up to the initialize step](./README-PNL.md#initialize)
+
+
+# Install Dependencies
+
+
+
+The following are the external dependencies required to run dpdash.
+
+## MongoDB
+
+To install mongodb manually follow the instructions [listed here](https://www.mongodb.com/docs/manual/installation/).
+
+A simpler way to install mongodb is through homebrew.
+
+```sh 
+brew tap mongodb/brew
+```
+```sh
+brew update
+```
+```sh
+brew install mongodb-community@5.0
+```
+
+Homebrew will start the services automatically for you. For more information on installing mongodb through homebrew follow the instructions [here](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-os-x/).
+
+## Rabbitmq 
+
+Rabbitmq is also used in this project and it must run on your machine. A generic build for UNIX with instructions on how to install is found [here](https://www.rabbitmq.com/install-generic-unix.html).
+
+There is a homebrew formulae to install rabbit mq. 
+To install it through homebrew follow these steps or click [here](https://www.rabbitmq.com/install-homebrew.html)
+
+```sh
+brew update
+```
+```sh
+brew install rabbitmq
+```
+
+## Prepare the Configs:
+- Copy the contents from the dashboard directory.
```

Comment:
We should probably be consistent with the use of `-` or `*` for bullets.

```suggestion
## Prepare the Configs:
* Copy the contents from the dashboard directory.
```

Reply:
[89cb808](https://github.com/AMP-SCZ/dpdash/pull/71/commits/89cb8080debb06653b9dca6dce4f49a2bb6260a8)